cmd/system-probe/modules/network_tracer.go

```diff
@@ -72,6 +72,9 @@ var NetworkTracer = module.Factory{
 
 		return &networkTracer{tracer: t, done: done}, err
 	},
+	NeedsEBPF: func() bool {
+		return runtime.GOOS != "windows"
```

Comment:
I would actually make this just specific to linux.

Reply:
Done in my last commit by making the module.Factory struct definition platform-specific.

Reply:
Oh I totally misunderstood your comment then.
In the current state of the PR, there's no need to check for the `runtime.GOOS` value but this means there's two declarations of the `NetworkTracer` and `EventMonitor` module factories, one for Linux and one for Windows. Are you ok with this or would you rather revert to having only one?